class-components/src/App.tsx

```diff
@@ -1,16 +1,40 @@
 import { StrictMode } from 'react';
-import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { RouterProvider } from 'react-router-dom';
 import ErrorBoundary from './components/ErrorBoundary/ErrorBoundary.tsx';
-import { routes } from './router.tsx';
+import { browserRouter } from './router.tsx';
+import { useSelectedItemsStore } from './store/selectedItemsStore.ts';
+import Flyout from './components/Flyout/Flyout.tsx';
+import { convertToCSV, downloadCSV } from './utils/csvConverter.ts';
 
-const browserRouter = createBrowserRouter(routes);
+const App = () => {
+  const { selectedPokemons, unselectAll } = useSelectedItemsStore();
```

Comment:
Just wondering if we can use this hook directly inside `Flyout`, istead of using it in the App file and passing the values down as props to Flyout? Would this work?